test/ecto/query/planner_test.exs

```diff
@@ -16,9 +16,49 @@ defmodule Ecto.Query.PlannerTest do
       field :temp, :string, virtual: true
       field :posted, :naive_datetime
       field :uuid, :binary_id
+      field :special, :boolean
       belongs_to :post, Ecto.Query.PlannerTest.Post
       has_many :post_comments, through: [:post, :comments]
     end
+
+    def special() do
```

Comment:
Special isn't the best name, a name that describes WHAT is special would be better.

Reply:
Its defined on the `Comment` schema, so `Comment.special()` returns a query for special comments.